test/stat_tracker_test.rb

```diff
@@ -112,6 +112,30 @@ def test_it_can_determine_team_with_worst_winning_percentage
     assert_equal 14, @stats.worst_team("20142015")
   end
 
+  def test_it_can_create_array_of_all_team_ids
+    expected = [1, 4, 6, 14, 26]
+    assert_equal expected, @stats.team_ids
+  end
+
+  def test_it_can_organize_season_win_percentage_for_each_team ###
+    expected = {
+      1 => 18.75,
+      4 => 31.25,
+      6 => 43.75,
+      14 => 12.5,
+      26 => 37.5
+    }
+    assert_equal expected, @stats.all_teams_win_percentage("20142015")
+  end
```

Comment:
We need to come back to this method to ensure we are getting the proper win percentages